.github/workflows/Nightly_Perf_Env_CI.yml

```diff
@@ -175,7 +175,7 @@ jobs:
         WORKER_DISK_IDS: ${{ secrets.PERF_WORKER_DISK_IDS }}
         TIMEOUT: 8000
         SCALE: 2
-        BOOTSTROM_SCALE: 100
+        BOOTSTROM_SCALE: 80
```

Comment:
While you're at it, please fix `BOOTSTROM_SCALE` to `BOOTSTORM_SCALE`.

Reply:
Thx done